Chaincase.UI/ViewModels/OverviewViewModel.cs

```diff
@@ -141,16 +150,16 @@ private async Task TryRewriteTableAsync()
 
         private async Task LoadWalletAsync()
         {
-            string walletName = _global.Network.ToString();
-            KeyManager keyManager = _global.WalletManager.GetWalletByName(walletName).KeyManager;
+            string walletName = _config.Network.ToString();
+            KeyManager keyManager = _walletManager.GetWalletByName(walletName).KeyManager;
             if (keyManager is null)
             {
                 return;
             }
 
             try
             {
-                _global.Wallet = await _global.WalletManager.StartWalletAsync(keyManager);
+                _global.Wallet = await _walletManager.StartWalletAsync(keyManager);
```

Comment:
maybe global.Wallet should be moved to walletmanager.CurrentWallet?

Reply:
Good Idea maybe we extend WalletManager for this chaincase-specific application. I like it.

Reply:
done